src/features.ts

```diff
@@ -1,7 +1,7 @@
 import env from 'src/utils/useEnvironment';
 import { CachedUserService } from 'src/services';
 
-export type FeatureName = 'Show Production View' | 'Projects' | 'User Detailed Sites';
+export type FeatureName = 'Show Production View' | 'Projects' | 'User Detailed Sites' | 'Deliverables MVP';
```

Comment:
I can't seem to list these on separate lines, the formatter brings them back to the same line.